CHANGELOG.md

```diff
@@ -11,8 +11,11 @@
 * Add option to add import statements at the top of the generated swift file (for
   storyboards) using the `import` flag. Multiple imports can be added by repeating
   this flag.  
-  [David Jennes](https://github.com/djbe), [#175](https://github.com/AliSoftware/SwiftGen/pull/175) 
+  [David Jennes](https://github.com/djbe), [#175](https://github.com/AliSoftware/SwiftGen/pull/175)
 * Remove the `key` param label from the `tr` function for Localized String in the swift3 template [AndrewSB](https://github.com/AndrewSB), [#190](https://github.com/AliSoftware/SwiftGen/pull/190)
+* The `swiftgen images` command now uses the `actool` utility to parse asset catalogs,
+  ensuring that the parser correctly handles namespaced folders.
```

Comment:
There typical expected format is the one used in 3.0.1 section and before.

```
* Description of the change.••
  [Author Name](GH profile link)
  [#PRNum](link to PR)
```
With `•` to be replaced by a space, otherwise going on a new, indented line for every link after the description because I have OCD 😂